webcomponents/tool/src/main/frontend/packages/sakai-lti-popup/src/SakaiLTIPopup.js

```diff
@@ -20,6 +20,7 @@ export class SakaiLTIPopup extends SakaiElement {
     this.preLaunchText = null;
     this.postLaunchText = null;
```

Comment:
The way you are testing for these being null in the i18n promise sets up a race condition. The setting of attributes happens after the constructor, but that setting may happen before the i18n promise fulfils. So, a user of the tag may think they are overriding the i18n default, but they maybe won't.